include/RVSDG/RVSDGOps.td

```diff
@@ -151,26 +151,26 @@ def RVSDG_LambdaResult: RVSDG_Op<"lambdaResult", [
     let hasVerifier = 1;
 }
 
-def RVSDG_ApplyNode: RVSDG_Op<"applyNode", [DeclareOpInterfaceMethods<CallOpInterface>]>{
-    let summary = "RVSDG apply node. Used to call lambda nodes";
-    let description = [{
-        RVSDG apply node. Takes a reference to the lambda node to be called as its first input
-        followed by the arguments for the lambda. Outputs the results of applying the lambda node.
-    }];
+// def RVSDG_ApplyNode: RVSDG_Op<"applyNode", [DeclareOpInterfaceMethods<CallOpInterface>]>{
+//     let summary = "RVSDG apply node. Used to call lambda nodes";
+//     let description = [{
+//         RVSDG apply node. Takes a reference to the lambda node to be called as its first input
+//         followed by the arguments for the lambda. Outputs the results of applying the lambda node.
+//     }];
```

Comment:
Why are these lines commented out? Should they be removed?

Reply:
Yes, apply is renamed to call and moved to jlm dialect